mongo/database.go

```diff
@@ -350,24 +384,33 @@ func (db *Database) ListCollectionNames(ctx context.Context, filter interface{},
 	return names, nil
 }
 
-// ReadConcern returns the read concern of this database.
+// ReadConcern returns the read concern used to configure the Database object.
 func (db *Database) ReadConcern() *readconcern.ReadConcern {
 	return db.readConcern
 }
 
-// ReadPreference returns the read preference of this database.
+// ReadPreference returns the read preference used to configure the Database object.
 func (db *Database) ReadPreference() *readpref.ReadPref {
 	return db.readPreference
 }
 
-// WriteConcern returns the write concern of this database.
+// WriteConcern returns the write concern used to configure the Database object.
 func (db *Database) WriteConcern() *writeconcern.WriteConcern {
 	return db.writeConcern
 }
 
-// Watch returns a change stream cursor used to receive information of changes to the database. This method is preferred
-// to running a raw aggregation with a $changeStream stage because it supports resumability in the case of some errors.
-// The database must have read concern majority or no read concern for a change stream to be created successfully.
+// Watch returns a change stream for all changes to the corresponding database. See
+// https://docs.mongodb.com/manual/changeStreams/ for more information about change streams.
+//
+// The Database must be configured with read concern majority or no read concern for a change stream to be created
+// successfully.
+//
+// The pipeline parameter should be an array of documents, each representing a pipeline stage. The pipeline or any of
```

Comment:
Should we say that the pipeline may be empty (here and elsewhere)? I'm not sure that it's obvious.

Reply:
I think that's a good idea. Added a note to Aggregate and Watch to say the pipeline can be empty.